lib/Storage/storage_managment.dart

```diff
@@ -0,0 +1,134 @@
+import 'dart:convert';
+import 'package:flagship/flagship.dart';
+import 'package:flagship/utils/logger/log_manager.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:path/path.dart';
+
+// Trying to add database import
+
+String lastModfiedKey = "FSLastModifiedScript";
+String fileName = "cacheHits.json";
+
+class DataBaseManagment {
```

Comment:
typo on the name: `DataBaseManagement`.
Also, it does not match the file name, and I would prefer `Database` instead of `DataBase`, because database is a whole word

Reply:
👍